dev/bots/docs.sh

```diff
@@ -139,7 +139,7 @@ if [[ -n "$CIRRUS_CI" && -z "$CIRRUS_PR" ]]; then
     echo "Updating $CIRRUS_BRANCH docs: https://docs.flutter.io/"
```

Comment:
```suggestion
    echo "Updating $CIRRUS_BRANCH docs: https://docs.flutter.dev/"
```
And above too. GitHub won't let me comment on that line.